packages/eslint-plugin/lib/config/testing-library.ts

```diff
@@ -1,22 +1,22 @@
-import { reactTestFiles, testFiles } from "../utils/patterns";
-
 import type { Linter } from "eslint";
+import testingLibraryPlugin from "eslint-plugin-testing-library";
+import { reactTestFiles, testFiles } from "../utils/patterns.ts";
 
-const config: Linter.Config = {
-    overrides: [
-        {
-            files: reactTestFiles,
-            plugins: ["testing-library"],
-            extends: ["plugin:testing-library/react"]
+const config: Linter.FlatConfig[] = [
```

Comment:
I think it's supposed to use 2 different extends:

Here's the original config:

```
const config: Linter.Config = {
    overrides: [
        {
            files: reactTestFiles,
            plugins: ["testing-library"],
            extends: ["plugin:testing-library/react"]
        },
        {
            files: testFiles,
            extends: ["plugin:testing-library/dom"]
        }
    ]
};
```

Reply:
I'll check, probably just missed the DOM one.

Reply:
The DOM rules are present in the second object.